kotlin-spark-api/3.2/pom_2.12.xml

```diff
@@ -14,6 +14,19 @@
   </parent>
   <packaging>jar</packaging>
 
+  <repositories>
+    <repository>
+      <id>kotlinx-html</id>
+      <name>kotlinx-html</name>
+      <url>https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven</url>
+    </repository>
+    <repository>
+      <id>kotlin</id>
+      <name>kotlin</name>
+      <url>https://maven.pkg.jetbrains.space/kotlin/p/kotlin/dev</url>
+    </repository>
+  </repositories>
+
```

Comment:
Looks like this change should not be a part of release because pacjages in central should not depend on other repositories

Reply:
How can we use kotlinx-html then? We could use another library or do it manually, but kotlinx-html is great :/

Reply:
If it's unavoidable — let's use it. ALso I think we should ask the team to publish to central. There is a huge risk corporate users won't be able to use it (and major part of jupyter customers are corporate)